Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA_test.py

```diff
@@ -0,0 +1,287 @@
+import json
+import io
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
```

Comment:
Unit tests should contain given-when-then comment to describe the unit test scenario. Given <some input>, When <some state> Then <ensure some behavior>.
This comment applies to all unit tests below. Please make sure to go over all of them and add the missing details.

Reply:
Resolved, Check-in next commit